docs/js-guide-bg-scripts.md

```diff
@@ -0,0 +1,58 @@
+---
+id: aragonjs-guide-bg-scripts
+title: aragonAPI - Background Scripts guide 
```

Comment:
what do you think about removing the word "guide"?  It isn't used anywhere else in hack.aragon and I don't think it's needed.